WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -1567,6 +1569,10 @@ public void onNegativeClicked(@NonNull String instanceTag) {
         }
     }
 
+    @Override public void onSetPromptReminderClick(final int siteId, final boolean isFirstTimePublishing) {
+        mBloggingRemindersViewModel.onPublishingPost(siteId, isFirstTimePublishing);
```

Comment:
Thanks for catching that! 
> We should probably add a dedicated method for this (eg. onBloggingPromptSchedulingRequested or something like this), instead of relying on onPublishingPost.

Makes sense 👍